bin/public/index.html

```diff
@@ -1,17 +1,111 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
```

Comment:
Is it possible to separate these files (Wizard of OS) into another directory? I can take care of loading them if some special option is provided.